### PR TITLE
[2017.7.9] Fix lint issues

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
Fix lint issues on 2017.7.9

### Tests written?

No

### Commits signed with GPG?

Yes

### CODE REVIEW

File: salt/pillar/git_pillar.py [hunk 1]
```diff
@@ -518,7 +518,7 @@ def __virtual__():
                     'could not be loaded, is GitPython installed?'
                 )
                 return False
-            if not git.__version__ > '0.3.0':
+            if git.__version__ <= '0.3.0':
```
Comment (waynew) [inline salt/pillar/git_pillar.py]: I know it's arguably unrelated to this PR, but... do we not have a better way to check version strings? I don't know if git guarantees it's version #s will be 0-9, and only 3 segments, but this seems mildly sketchy to me.

Comment (waynew) [inline salt/pillar/git_pillar.py]: I know it's arguably unrelated to this PR, but... do we not have a better way to check version strings? I don't know if git guarantees it's version #s will be 0-9, and only 3 segments, but this seems mildly sketchy to me.

Comment (s0undt3ch) [reply salt/pillar/git_pillar.py]: Yes, unrelated to this PR ;)
And, to add up, this kind of comparison works on py2 and not that well on py3.

**Comment by waynew** [comment]
Plot twist: this breaks some Whitespace program :trollface:

**Review by waynew — APPROVED** [review]

**Pull request closed without merging**